pg_service_parser/core/pg_service_parser_wrapper.py

```diff
@@ -82,8 +91,13 @@ def copy_service_settings(source_service_name: str, target_service_name: str, co
     assert service_config("qgis-demo") == _settings
 
     # Add new service
-    _settings = {'host': 'localhost', 'port': '5433', 'user': 'admin', 'password': 'secret',
-                 'dbname': 'qgis_test_db2'}
+    _settings = {
+        "host": "localhost",
+        "port": "5433",
```

Comment:
```suggestion
        "port": "5432",
```
?

Reply:
Those are/were just quick tests. There I was changing parameters so that the overwrite result was even more evident.

BTW, those quick tests could eventually become unit tests in this repo.